web/cypress/integration/route/create-route-with-chash-upstream.spec.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Edit Route With Custom CHash Key Upstream', () => {
+  const selector = {
+    name: '#name',
+    upstreamType: ".ant-select-item-option-content",
+    hashPosition: ".ant-select-item-option-content",
+    nodes_0_host: '#nodes_0_host',
+    nodes_0_port: '#nodes_0_port',
+    nodes_0_weight: '#nodes_0_weight',
+    nameSelector: '[title=Name]',
+    chash_key: '#key',
+    deleteAlert: '.ant-modal-body',
+    notificationCloseIcon: '.ant-notification-close-icon',
+    notification: '.ant-notification-notice-message',
+  };
+
+  const data = {
+    routeName: 'roteName',
+    ip: '127.0.0.1',
+    port: '7000',
+    weight: '1',
+    deleteRouteSuccess: 'Delete Route Successfully',
+    submitSuccess: 'Submit Successfully',
+    custom_key: 'custom_key',
+    new_key: 'new_key',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should create route with custom chash key Upstream', function () {
+    cy.visit('/');
+    cy.get('[role=menu]')
```

Comment:
```suggestion
    cy.get('selector.XXX')
```
It is best to keep the context consistent when you use `selector.xxx`

Reply:
Thank you for the guidance of the code standards. Please help me review again. @guoqqqi